Kommunicate-io/Kommunicate-Web-SDK
# FW-3312 Updated message preview logic

### What do you want to achieve?
- add support for various rich messages in the preview

### PR Checklist


### How was the code tested?

- get an account having all types of rich messages configured in a kompose bot
- launch a widget
- open a conversation with the bot
- type in the intent trigger and close the widget immediately to see the message preview
- check for all the rich messages, attachments etc
- 

### What new thing you came across while writing this code? 
-

### In case you fixed a bug then please describe the root cause of it? 
-

NOTE: Make sure you're comparing your branch with the correct base branch
